client/driver/fabfile.py

```diff
@@ -663,6 +663,10 @@ def _http_content_to_json(content):
     return json_content, decoded
 
 
+def _parse_bool(val):
```

Comment:
This function is now located in utils.py: https://github.com/cmu-db/ottertune/blob/master/client/driver/utils.py#L50